server/py/framework/db/sqldb/db.py

```diff
@@ -3191,6 +3206,79 @@ def _calculate_runs_counters(
             project_to_running_runs_count,
         )
 
+    def _calculate_alerts_activations_counters(
+        self,
+        session,
+    ) -> tuple[
+        dict[str, int],
+        dict[str, int],
+        dict[str, int],
+    ]:
+        projects_with_creation_time = self.list_projects(
+            session,
+            format_=mlrun.common.formatters.ProjectFormat.name_and_creation_time,
+        ).projects
```

Comment:
It is not needed to request projects here, because we already do it here https://github.com/mlrun/mlrun/blob/8b5aec2a398611d99518c43d879cf88be95a8117/server/py/services/api/crud/projects.py#L440-L444. Format can be just changed from `name_only` to `name_and_creation_time` - this way instead of doing 2 request to db, we will do just one. And then project with creation time can be passed to this function (the same as we do with `list_alert_activations`)